man/man1/kbsecret-session.1.ronnpp

```diff
@@ -0,0 +1,90 @@
+kbsecret-session(1) - create a new kbsecret(1) session
+=======================================================
+
+## SYNOPSIS
+
+`kbsecret session new` [options] <label>
+`kbsecret session rm` [options] <session>
+
+## DESCRIPTION
+
+Creates or deletes kbsecret sessions.
+
+## COMMANDS
+
+* `new`:
+	Creates a new session.
+
+	There are two kinds of sessions: *team-based* and *teamless* (the default).
```

Comment:
This language can go in the `DESCRIPTION`, since it applies to sessions in general.